var/spack/repos/builtin/packages/fujitsu-ssl2/package.py

```diff
@@ -142,3 +143,12 @@ def setup_dependent_build_environment(self, env, dependent_spec):
         env.append_flags(
             "FCC_ENV", "-idirafter " + path
         )
+
+    @property
+    def headers(self):
+        path = join_path(
+            self.spec.prefix, "clang-comp"
+        )
+        headers = find_headers('cssl', path, recursive=True)
+        headers.directories = os.path.dirname(headers[0])
```

Comment:
Is this line needed, or does the default `directories` attribute work?

Reply:
Thank you for your advice . 
I removed unnecessary lines according to your advice .